[2023-06-27T01:38:46.933+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_dag.inaccurate scheduled__2023-06-26T00:00:00+00:00 [queued]>
[2023-06-27T01:38:47.130+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_dag.inaccurate scheduled__2023-06-26T00:00:00+00:00 [queued]>
[2023-06-27T01:38:47.137+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-06-27T01:38:47.424+0000] {taskinstance.py:1327} INFO - Executing <Task(BashOperator): inaccurate> on 2023-06-26 00:00:00+00:00
[2023-06-27T01:38:47.614+0000] {standard_task_runner.py:57} INFO - Started process 77 to run task
[2023-06-27T01:38:47.746+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'my_dag', 'inaccurate', 'scheduled__2023-06-26T00:00:00+00:00', '--job-id', '67', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmpztr_vk11']
[2023-06-27T01:38:47.878+0000] {standard_task_runner.py:85} INFO - Job 67: Subtask inaccurate
[2023-06-27T01:38:48.876+0000] {task_command.py:410} INFO - Running <TaskInstance: my_dag.inaccurate scheduled__2023-06-26T00:00:00+00:00 [running]> on host 414be138234a
[2023-06-27T01:38:51.243+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='my_dag' AIRFLOW_CTX_TASK_ID='inaccurate' AIRFLOW_CTX_EXECUTION_DATE='2023-06-26T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-06-26T00:00:00+00:00'
[2023-06-27T01:38:51.264+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-06-27T01:38:51.276+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo inaccurate']
[2023-06-27T01:38:51.407+0000] {subprocess.py:86} INFO - Output:
[2023-06-27T01:38:51.421+0000] {subprocess.py:93} INFO - inaccurate
[2023-06-27T01:38:51.425+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-06-27T01:38:51.871+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=my_dag, task_id=inaccurate, execution_date=20230626T000000, start_date=20230627T013846, end_date=20230627T013851
[2023-06-27T01:38:52.127+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-06-27T01:38:52.244+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
